modules/javafx.graphics/src/test/java/test/com/sun/javafx/test/binding/ReflectionHelper.java

```diff
@@ -42,10 +42,8 @@ private ReflectionHelper() {
 
     public static Object newInstance(final Class<?> cls) {
         try {
-            return cls.newInstance();
-        } catch (final InstantiationException e) {
-            throw convertToRuntimeException(e);
-        } catch (final IllegalAccessException e) {
+            return cls.getDeclaredConstructor().newInstance();
+        } catch (final Exception e) {
```

Comment:
This could re-wrap a `RuntimeException` needlessly, but is OK in this case (it's a test and it won't change whether or not it fails). The more equivalent fix would be to either list the checked exception individually, or to add a `catch (RuntimeException)` before the `catch (Exception)` and re-throw the rte without wrapping it.

Reply:
Avoided the re-wrap of a `RuntimeException` as suggested.